dts/bindings/display/solomon,ssd1322.yaml

```diff
@@ -0,0 +1,10 @@
+description: SSD1322 display controller
+
+compatible: "solomon,ssd1322"
+
+include: [mipi-dbi-spi-device.yaml, display-controller.yaml]
+
+properties:
+  column_offset:
+    type: int
+    description: First visible column number, default 0.
```

Comment:
If there is a default value, should we add `default` key here so that the user does not have to specify `column_offset = <0>`?

Also:
- Please add a justification for the default. Can be as simple as "reset setting for the controller" if this is the case
- property names should be styled like "column-offset"

Reply:
I have rethought that and providing a default value here maybe is not the best option. I guess that this value is different for each display model, so there is not a default one. I have change this property to be required.